### PR TITLE
fix: command options and choices doesn't syncs

### PR DESCRIPTION
**Pull request by Damego**
## About

This pull request fixes bug command options/choices doesn't syncs when you tryies to add/update/remove options/choice.

Reason of this bug is in overriding `_command` variable on line 259.
Also this pr removes useless return data

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues) (If existent):.
  - resolves #
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

**Review by sillyElvy — CHANGES_REQUESTED** [review]
Was this tested properly and with enough commands? I remember having issues before adding that

**Comment by Damego** [comment]
> Was this tested properly

I adds, updates, deletes options and choices in all kinds(maybe?) of commands(groups and subcommands) and it works fine.

File: interactions/client/bot.py [hunk 1]
```diff
@@ -301,8 +300,7 @@ def __check_options(command, data):
 
                         elif command.get("options") and data.get("options"):
 
-                            clean, _command = __check_options(command, data)
-
+                            clean = __check_options(command, data)
```
Comment (Damego) [inline interactions/client/bot.py:303]:
```suggestion
                            clean = __check_options(command, data)

```

**Review by sillyElvy — APPROVED** [review]
If it works it works, yay

**Pull request merged**